public/js/reader.js

```diff
@@ -64,6 +65,13 @@ const readerComponent = () => {
 
 					const savedFlipAnimation = localStorage.getItem('enableFlipAnimation');
 					this.enableFlipAnimation = savedFlipAnimation === null || savedFlipAnimation === 'true';
+
+					const savedRightToLeft = localStorage.getItem('enableRightToLeft');
+					if (savedRightToLeft === null) {
+						this.enableRightToLeft = false;
+					} else {
+						this.enableRightToLeft = savedRightToLeft;
```

Comment:
Good catch :+1:

Reply:
Ah, of course. It's been a while since I used a dynamic language. I've swapped to checking with `(savedRightToLeft === 'true')` instead.

This also looks like it's fixed the checkbox behavior that hkalexling mentioned. I suspect because `Boolean('false')` returns true.